File: app/src/main/java/com/example/androidviewdemo/MainActivity.java
package com.example.androidviewdemo;

import androidx.appcompat.app.AppCompatActivity;

import android.content.Intent;
import android.os.Bundle;
import android.view.View;

public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        findViewById(R.id.jump_page).setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                Intent intent = new Intent(MainActivity.this, ClassSevenTestActivity.class);
                startActivity(intent);
            }
        });
    }
}

File: hencoderl14_viewpager/src/main/java/com/example/hencoderl14_viewpager/MainActivity.kt
package com.example.hencoderl14_viewpager

import android.content.Intent
import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle

class MainActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
        val intent  = Intent(this, ViewPagerActivity::class.java)
        startActivity(intent)
    }
}
